mne/datasets/utils.py

```diff
@@ -235,7 +235,7 @@ def _data_path(path=None, force_update=False, update_path=True, download=True,
     path = _get_path(path, key, name)
     # To update the testing or misc dataset, push commits, then make a new
     # release on GitHub. Then update the "releases" variable:
-    releases = dict(testing='0.46', misc='0.3')
+    releases = dict(testing='0.47', misc='0.3')
```

Comment:
The release you need is `0.48` with hash `819c0094f03d494b340d5778cd383a8f`

Reply:
thanks!